.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "rules": {
+    "semi": [2, "never"],
+    "prefer-arrow-callback": 0,
+    "space-before-function-paren": [2, "never"],
+    "func-names": 0,
+    "no-param-reassign": 0,
```

Comment:
The rule references [this article](http://eslint.org/docs/rules/no-param-reassign), which gives a great case as to why this is a bad idea. That being said, at least according to [this article](https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#3-managing-arguments) (which you linked me :stuck_out_tongue:), if you don't _mention_ the `arguments` variable, and are in `"use strict"`, then you should be safe to disable this rule.

I personally prefer to avoid the potential issue, but as long as you are careful this should be fine to disable.